buildlib/dockers/docker-compose.yml

```diff
@@ -12,6 +12,17 @@ services:
         MOFED_OS: rhel7.6
         CUDA_VERSION: 11.2.0
         OS_VERSION: 7
+  centos7-gpu-mofed5-cuda11:
+    image: centos7-gpu-mofed5-cuda11
+    build:
+      context: .
+      network: host
```

Comment:
pls comment that it's temporary solution to not disturb non-docker tests

Reply:
No, this is not applicable to CI, azure will run docker on its network 
it need more for manial launch

Reply:
so you plane to remove it?

Reply:
No